### ISSUE TITLE
No CRLF added to output files (which is needed for parsing logfile)

### ISSUE THREAD

**Issue by voidnecron** [skelsec/pypykatz#46]
Mimikatz logfiles, obviously created on Windows machine add a CRLF line terminator when being created.
Pypykatz does not, which causes issues with other applications/scripts when parsing the logfiles.
Doing a 'file <log>' gives you this difference:
lsass-mimi.log:  ASCII text, with very long lines, with CRLF line terminators
lsass-pypy.log: ASCII text, with very long lines

Perhaps pypykatz could use the print function differently so it writes files with CRLF?
Now I have to do 'recode CR-LF <log>', which isn't that hard, but if not needed it would be nice.

https://www.techwalla.com/articles/how-to-use-echo-in-python

**Comment by skelsec** [comment]
I do not see the reason why you want to parse the "logfiles" (assuming you mean the default text output), can you clarify?
I have added ```--json``` and ```--grep``` output options specifically for parsing the output programmatically. Is there a problem parsing those?

**Comment by voidnecron** [comment]
TL;DR: Pypykatz output is a different syntax as Mimikatz (For instance NT: <hash> for pypykatz; * NTLM: <hash> for mimikatz) and therefor my regex fails for extracting needed parts (username, password, domain, NTLM hash). 

I dug a little deeper and found it wasn't (only) because of the missing CRLF lines, but the output created by Pypykatz is different from Mimikatz.
Reason I do this is to automate the process as much as possible, by dropping a lsass memory dumpfile a cronjob will run which executes pypykatz (lsa minidump $inputfile -o $outfile).
After this a second part will execute, the parser, which extracts usernames with cleartext credentials in one file, and usernames with NTLM hashes in a second file. The cleartext credentials will be put in the dictionary file. The second file will be fed to Hashcat to be cracked.

**Comment by skelsec** [comment]
The grep flag does show up in the help. ```pypykatz lsa -h ```
The normal text output was not meant for automatic parsing, JSON output is the one that has all the information in a structured manner -except for the kerberos tickets-, please use that.
pypykatz's output was never intended to mimic minikatz's output.

**Comment by voidnecron** [comment]
Thanks for your patience and explanations! Appreciate your time and effort!
